docs/releases/2.0.txt

```diff
@@ -194,6 +194,8 @@ Management Commands
 
 * The new :option:`diffsettings --output` option allows formatting the output
   in a unified diff format.
+* On Oracle, inspectdb can now introspect ``AutoField``, if the column is
```

Comment:
add backticks around inspectdb and add a blank line above.
chop the comme before "if"